mlrun/datastore/store_resources.py

```diff
@@ -0,0 +1,128 @@
+# Copyright 2018 Iguazio
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# flake8: noqa  - this is until we take care of the F401 violations with respect to __all__ & sphinx
+
+import mlrun
+from mlrun.config import config
+from mlrun.utils.helpers import parse_function_uri
+from .v3io import v3io_path
+from ..utils import StorePrefix, parse_store_uri
+
+
+class ResourceCache:
+    """Resource cache for real-time pipeline/serving and storey"""
```

Comment:
I would add a warning here that this cache does not have stuff like ttl, or other logic to prevent itself from going out of sync therefore it should be used only in certain scenarios where this is ok

Reply:
i think its quite obvious from the code and its for internal use, but added a comment to make u happy :)